src/js/_enqueues/wp/customize/controls.js

```diff
@@ -1264,11 +1266,14 @@
 		 * @return {void}
 		 */
 		_animateChangeExpanded: function( completeCallback ) {
-			// Return if CSS transitions are not supported.
-			if ( ! normalizedTransitionendEventName ) {
-				if ( completeCallback ) {
-					completeCallback();
-				}
+			// Return if CSS transitions are not supported or if reduced motion is enabled.
+			if ( ! normalizedTransitionendEventName || isReducedMotion ) {
+				// Schedule the callback until the next tick to prevent focus loss.
+				_.defer( function () {
```

Comment:
Where was the focus loss occurring?

Reply:
It should focus the "Back" button when expanding to a section. That is lost if we call the callback synchronously though.